optimum/exporters/openvino/model_configs.py

```diff
@@ -3873,20 +3880,28 @@ class M2M100OpenVINOConfig(BartOpenVINOConfig):
     pass
 
 
-@register_in_tasks_manager(
-    "deepseek_v3", *["text-generation", "text-generation-with-past"], library_name="transformers"
-)
 @register_in_tasks_manager(
     "deepseek_v2", *["text-generation", "text-generation-with-past"], library_name="transformers"
 )
 @register_in_tasks_manager("deepseek", *["text-generation", "text-generation-with-past"], library_name="transformers")
 class DeepseekOpenVINOConfig(MiniCPM3OpenVINOConfig):
+    MAX_TRANSFORMERS_VERSION = "4.54.0"
+
     def patch_model_for_export(
         self, model: Union["PreTrainedModel", "TFPreTrainedModel"], model_kwargs: Optional[Dict[str, Any]] = None
     ) -> "ModelPatcher":
         return DeepseekPatcher(self, model, model_kwargs=model_kwargs)
 
 
+@register_in_tasks_manager(
+    "deepseek_v3", *["text-generation", "text-generation-with-past"], library_name="transformers"
+)
+class DeepseekVOpenVINOConfig(LlamaOpenVINOConfig):
+    MIN_TRANSFORMERS_VERSION = "4.54.0"
```

Comment:
I guess it was added earlier

Reply:
actually removed in https://github.com/huggingface/optimum-intel/pull/1406/commits/8dea7ebc62ba3d5f5e906974deedd1e86125dc7c as it seems to be an issue (output between original and openvino models differs in some cases likely coming from https://github.com/huggingface/transformers/blob/main/src/transformers/models/deepseek_v3/modeling_deepseek_v3.py#L409) : same issue as observed for ernie 4.5 cc @openvino-dev-samples

Reply:
so setting MAX_VERSION_TRASNFORMERS in this PR (as the model defined in https://huggingface.co/deepseek-ai/DeepSeek-V3.1-Base/blob/main/modeling_deepseek.py is not compatible with transformers v4.54 and above) and will add deepseek v3 support from transformers https://github.com/huggingface/transformers/blob/main/src/transformers/models/deepseek_v3/modeling_deepseek_v3.py in a following PR (along with ernie 4.5 as issue is the same)